com.unity.formats.fbx/Editor/FbxExporter.cs

```diff
@@ -3136,14 +3136,19 @@ int objectCount
         /// components and the animation clips. Also, the first animation to export, if any.
         /// </summary>
         /// <returns>The animation only hierarchy count.</returns>
-        /// <param name="exportSet">GameObject hierarchies selected for export.</param>
         /// <param name="hierarchyToExportData">Map from GameObject hierarchy to animation export data.</param>
         [SecurityPermission(SecurityAction.LinkDemand)]
         internal int GetAnimOnlyHierarchyCount(Dictionary<GameObject, IExportData> hierarchyToExportData)
         {
             // including any parents of animated objects that are exported
             var completeExpSet = new HashSet<GameObject>();
-            foreach (var data in hierarchyToExportData.Values) {
+            foreach (var item in hierarchyToExportData) {
+                var data = item.Value;
+                if(data == null || data.Objects.Count <= 0)
```

Comment:
Is `data` a `UnityEngine.object`?

If so we should test it like a boolean rather than test for null, so we can also catch "missing" values.

Reply:
data is an interface containing a HashSet of GameObjects:
```
 internal interface IExportData
 {
     HashSet<GameObject> Objects { get; }
 }
```